presentation/src/main/java/com/sopt/presentation/calendar/component/bottomsheet/ScheduleColorChip.kt

```diff
@@ -0,0 +1,32 @@
+package com.sopt.presentation.calendar.component.bottomsheet
+
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.size
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.clip
+import androidx.compose.ui.platform.LocalContext
+import androidx.compose.ui.unit.dp
+import com.sopt.core.designsystem.theme.NoostakTheme
+import com.sopt.core.type.CategoryType
+
+@Composable
+fun ScheduleColorChip(category: String) {
+    val context = LocalContext.current
+
+    Box(
+        modifier = Modifier
+            .size(width = 6.dp, height = 38.dp)
+            .clip(RoundedCornerShape(31.dp))
+            .background(
+                color = when (CategoryType.fromText(context, category)) {
+                    CategoryType.IMPORTANT -> NoostakTheme.colors.orange
+                    CategoryType.SCHEDULE -> NoostakTheme.colors.blue
+                    CategoryType.HOBBY -> NoostakTheme.colors.purple
+                    CategoryType.ETC -> NoostakTheme.colors.mint
+                }
+            )
+    )
+}
```

Comment:
P2: 여기 캘린더뷰랑 로직 겹치는 것 같네요!! 기존 확장함수 사용하면 좋을 것 같습니다!

```suggestion
@Composable
fun ScheduleColorChip(categoryType: CategoryType) {
    Box(
        modifier = Modifier
            .size(width = 6.dp, height = 38.dp)
            .clip(RoundedCornerShape(31.dp))
            .background(color = categoryType.toColor())
    )
}
```

Reply:
맞아요 안그래도 캘린더 pr 볼 때 머지하면 재사용하려고 생각했어요 적용했어요!